cmd/starsd/cmd/root.go

```diff
@@ -206,15 +202,15 @@ func (ac appCreator) newApp(
 	}
 
 	snapshotDir := filepath.Join(cast.ToString(appOpts.Get(flags.FlagHome)), "data", "snapshots")
-	snapshotDB, err := sdk.NewLevelDB("metadata", snapshotDir)
+	snapshotDB, err := dbm.NewDB("metadata", dbm.GoLevelDBBackend, snapshotDir) // COMEBACK HERE Unsure what backend to use- using goleveldb now
```

Comment:
anything we should consider here ?

Reply:
Instead of hardcoding to `dbm.GoLevelDBBackend`, should probably have something like
https://github.com/cosmos/cosmos-sdk/blob/875a71cc6f7c0c981f7d7a75ea4f33d93e45185d/server/util.go#L415-L431

So PebbleDB and RocksDB can be used as well